CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```

Comment:
🔥 
Devs typically know that this is the purpose of a `CHANGELOG.md`. They wouldn't be in this file otherwise.